### PR TITLE
[MRG] Move get_matched_empty_room to BIDSPath.find_empty_room()

### PR DESCRIPTION
**Pull request by hoechenberger**
PR Description
--------------

Closes GH-524

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/master/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

File: mne_bids/path.py [hunk 1]
```diff
@@ -10,18 +10,147 @@
...
+    reader)
 from mne_bids.utils import (_check_key_val, _check_empty_room_basename,
                             _check_types, param_regex,
                             _ensure_tuple)
 
 
+def _get_matched_empty_room(bids_path):
+    """Get matching empty-room file for an MEG recording."""
+    if not isinstance(bids_path, BIDSPath):
+        raise RuntimeError('"bids_path" must be a BIDSPath object. Please '
```
Comment (agramfort) [inline mne_bids/path.py:31]: This check is now unnecessary

**Review by agramfort — COMMENTED** [review]
Besides lgtm

File: mne_bids/path.py (hunk 1 above)
Comment (hoechenberger) [reply mne_bids/path.py]: Removed in 9a3875a

**Comment by agramfort** [comment]
Thx @hoechenberger

**Pull request merged**